### DISCUSSION TITLE
Hash in URL breaks Navigation

### DISCUSSION THREAD

**Discussion by monsonjeremy** [react-navigation/react-navigation#10736] [Q&A]
Is this a known issue with `react-navigation`:

I have an app that is using `supabase` and the verification email for user sign-up has a redirectUrl of the following form:
`exp://10.0.0.1:19000/--/verified#access_token=....&refresh_token=.....`

This URL breaks deep linking for me since it seems like `react-navigation` doesn't parse hashes like you would expect a browser to.

**Comment by NilsBaumgartner1994** [comment]
Yes facing the same problem. Hashtags are encoded to %23 which breaks my navigation too.
